### DISCUSSION TITLE
[Feature] Jump to last position in pictures and people's view

### DISCUSSION THREAD

**Discussion by jhf2442** [immich-app/immich#6052] [Feature Request]
### The feature

When clicking on one picture in picture view / on one face in peoples view, you come to the requested page.
From there, hitting either the back button of the browser or the back button at the top left of immich's UI, you come back to the picture view or people view, but always at the *beginning* of the list, no matter where you were before.

Request would be that immich memorizes the position in the list and jumps back there. This helps when assigning names in a list of hundreds of people

Discourse for instance achieves this by modifying dynamically the URL of the page to be .../threadID/postID when scrolling through the posts of a thread. Therefore the URL in immich should be expanded to be .../people/personID instead of just /people

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#3114
